lte/gateway/c/core/oai/include/messages_def.h

```diff
@@ -37,7 +37,7 @@
 #include "lte/gateway/c/core/oai/include/s11_messages_def.h"
 #include "lte/gateway/c/core/oai/include/s1ap_messages_def.h"
 #include "lte/gateway/c/core/oai/include/ngap_messages_def.h"
-#include "lte/gateway/c/core/oai/include/s6a_messages_def.h"
+#include "lte/gateway/c/core/oai/include/s6a_messages_def.hpp"
```

Comment:
Can we add #pragma once to this file as well

Reply:
"#pragma once" will not work for this file, because this file is included more than once in the file intertask_interface_types.h. I have added the same comment in this file as well.